tutorials/eth-api/chat-gpt.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Using GPT-4 to Write and Debug Solidity Smart Contracts
+description: Learn how you can use OpenAI's ChatGPT (GPT-4) generative AI LLM to write, debug, and deploy solidity smart contracts on the Moonbeam network.
+---
+
+# Using GPT-4 to Write and Debug Solidity Smart Contracts
+
+![Banner Image](/images/tutorials/eth-api/chatgpt/chatgpt-banner.png)
+
+_May 10, 2023 | by Kevin Neilson_
+
+## Introduction {: #introduction } 
+
+Today, it's near impossible to walk down the street and not overhear a conversation about the transformative impact of generative AI. Of course, this applies to both the physical and virtual world (e.g. Twitter). You've probably heard by now that artificial intelligence tools like ChatGPT can plan your vacation, draft an essay, and tell you a joke. But did you know that ChatGPT can even write working solidity code for you? And, it doesn't just return a `.sol` file to you without any context. It can actually explain to you how the code is structured, walk you through deployment steps, and *drumroll* even write your test files for you. Yup, that's right. No more excuses for lack of test coverage when ChatGPT can take care of that for you. 
+
+In this tutorial, we'll look at how ChatGPT can help you write, deploy, and debug solidity smart contracts. But first, let's dive a bit more into what ChatGPT is exactly. 
+
+## An Overview of ChatGPT {: #an-overview-of-chatgpt }
+
+[ChatGPT](https://chat.openai.com/){target=_blank} is a text-based large language model (LLM) created by the company OpenAI. According to OpenAI, *"The dialogue format makes it possible for ChatGPT to answer followup questions, admit its mistakes, challenge incorrect premises, and reject inappropriate requests."* ChatGPT can hold a conversation with you and remember your chat history until a new session is started. To learn more about ChatGPT, check out [this introduction to ChatGPT on the OpenAI Blog](https://openai.com/blog/chatgpt){target=_blank}.
+
+### GPT-4 vs. ChatGPT
```

Comment:
header attribute is missing here, please add here and to the Limitations section